config/database.yml

```diff
@@ -1,43 +1,45 @@
-# SQLite version 3.x
-#   gem install sqlite3
-#
-#   Ensure the SQLite 3 gem is defined in your Gemfile
-#   gem 'sqlite3'
-#
-default: &default
-  adapter: sqlite3
-  pool: 5
-  timeout: 5000
-
-development:
-  <<: *default
-  database: db/development.sqlite3
-
-#development:
-  #adapter: mysql2
-  #encoding: utf8
-  #database: db_name
-  #username: user
-  #password: pass
-  #timeout: 15
-  #pool: 30
-
-# development:
-#   adapter: postgresql
-#   encoding: unicode
-#   database: db_name
-#   host: localhost
-#   pool: 5
-#   username: user
-#   password: pass
-
-# Warning: The database defined as "test" will be erased and
-# re-generated from your development database when you run "rake".
-# Do not set this db to the same as development or production.
-test:
-  <<: *default
-  database: db/test.sqlite3
-
-production:
-  <<: *default
-  database: db/production.sqlite3
+# SQLite version 3.x
+#   gem install sqlite3
+#
+#   Ensure the SQLite 3 gem is defined in your Gemfile
+#   gem 'sqlite3'
+#
+# default: &default
+#   adapter: sqlite3
+#   pool: 5
+#   timeout: 5000
+#
+# development:
+#   <<: *default
+#   database: db/development.sqlite3
+
+development:
+  adapter: mysql2
+  encoding: utf8
+  database: my_db
+  username: root
+  password: 1234

```

Comment:
Don't let the world know about this :)

We either create an example file with the basic configuration, and the config file itself (this file) is not added to the repo, or the values go into .env file. Either way we do not expose our passwords online, plus this messes up my configuration as soon as we merge your changes.